Kudu.Core/K8SE/K8SEDeploymentHelper.cs

```diff
@@ -161,17 +161,41 @@ private static string RunBuildCtlCommand(string args, string msg)
 
         public static string GetAppName(HttpContext context)
         {
-            var appName = context.Request.Headers["K8SE_APP_NAME"].ToString();
-
+            var appName = context.Request.Headers["k8se_app_name"].ToString();
```

Comment:
Verified that the Header is case-insensitive, so we don't need this change

Reply:
yes, will revert these change.